conf/machine/include/imx8mm-evk.inc

```diff
@@ -0,0 +1,46 @@
+MACHINEOVERRIDES =. "mx8:mx8m:mx8mm:"
+
+require conf/machine/include/imx-base.inc
+require conf/machine/include/tune-cortexa53.inc
+
+MACHINE_FEATURES += "pci wifi bluetooth optee bcm43455 bcm4356"
+
+# NXP BSP can consume BCM4359 and QCA9377 driver and firmware
+# Since the firmware is not available publicly, and rather distributed
+# under "Proprietary" license - we opt-out from using it in all BSPs
+# and pin it to NXP BSP only
+MACHINE_FEATURES_append_use-nxp-bsp = " bcm4359 qca9377"
```

Comment:
If the `qca9377` has been dropped from _conf/machine/imx7ulpevk.conf_ as it is not supported - should this be dropped in here as well?

If none of machines appears to have a support for QCA modules, can the entire _recipes-bsp/firmware-qca_ folder be removed? I believe if nothing requires those modules anymore - there is no reason to keep those "stale" recipes in the layer.

@otavio Another suggestion for _recipes-bsp/firmware-qca_: can they be moved to _meta-freescale-3rdparty_ layer from here?

Those recipes seems to become stale after `MACHINE_FEATURES` would be adjusted here and they seems no longer be needed in core _meta-freescale_ layer.

But in order to keep those recipes for anyone still using them in separate machine derivatives - they could be transferred to 3rd-party layer.

Reply:
Yes, if we are not using it here anymore it is better to move it out.

Reply:
8M Mini EVK is the one and only board that does still support `qca9377`.

Reply:
> 
> 
> 8M Mini EVK is the one and only board that does still support `qca9377`.

I'm a bit confused here: as you've indicated in commit d2ad44a50d684b9c9fdcffcad48028ef060dddf4 - `qca9377` is _not_ supported since kernel 4.14.y, yet `imx8mmevk` supports it despite of the fact that it uses 5.4.y kernel release...

How this support is provided? Do you offer a backport of 4.14.y kernel modules to 5.4.y?

If that is the case, then this should be explicitly indicated as people might base their design with `qca9377` module included without knowing that there is no upstream support for it provided.

Besides, _recipes-bsp/firmware-qca_ also have a recipe for `qca6174`, which is not used anywhere anymore - that one can be moved to _meta-freescale-3rdparty_ layer. But in this case the include should be either moved or modified...

-- andrey

Reply:
> I'm a bit confused here: as you've indicated in commit d2ad44a - qca9377 is not supported since kernel 4.14.y, yet imx8mmevk supports it despite of the fact that it uses 5.4.y kernel release...

I meant in that commit message only that 7ULP support had been dropped. Should I update the commit message to make it clearer?

Reply:
> 
> 
> > I'm a bit confused here: as you've indicated in commit [d2ad44a](https://github.com/Freescale/meta-freescale/commit/d2ad44a50d684b9c9fdcffcad48028ef060dddf4) - qca9377 is not supported since kernel 4.14.y, yet imx8mmevk supports it despite of the fact that it uses 5.4.y kernel release...
> 
> I meant in that commit message only that 7ULP support had been dropped. Should I update the commit message to make it clearer?

Ah, OK - got it now. It would be really helpful if you clarify this in the commit message then, otherwise somebody else later might get confused the same way I was. 😉 

Additionally, do you know if both _DDR4_ and _LPDDR3_ EVK variants have `qca9377`? If it is only one of those two - then it is better to move `qca9377` machine feature to the machine config file from this include. By that it would be easier to move the _recipes-bsp/firmware-qca_ once the machine would go EOL.

-- andrey

Reply:
Sure, will update the commit message.

> Additionally, do you know if both DDR4 and LPDDR3 EVK variants have qca9377? 

Both do.